nifi-toolkit/nifi-toolkit-encrypt-config/pom.xml

```diff
@@ -30,8 +30,8 @@
             <version>1.14.0-SNAPSHOT</version>
         </dependency>
         <dependency>
-            <groupId>org.apache.nifi</groupId>
-            <artifactId>nifi-properties</artifactId>
+            <groupId>org.apache.nifi.registry</groupId>
+            <artifactId>nifi-registry-properties</artifactId>
```

Comment:
Is it intentional to replace `nifi-properties` here with `nifi-registry-properties`?  Should both dependencies be declared?

Reply:
Yes, since the CryptoKeyUtil (in nifi-registry-properties) is now used.   Since nifi-registry-properties depends on nifi-properties, I didn't include them both.  However, I agree it would be more clear to include both, so I'll add nifi-properties again.